.gitignore

```diff
@@ -11,3 +11,4 @@ MonitoringDemo.Sql/transport/
 .learningtransport/**
 
 **/.diagnostics/*
+/src/.learningtransport
```

Comment:
Would it make sense to simply change the above rule from

```
.learningtransport/**
````

to

```
**/.learningtransport/*
```

? Btw. I haven't tested if that works. Just typed it from memory